ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -273,21 +320,36 @@ private void OnImageStatusUpdate(Border view, ImageLoadStatus status, ImageMetad
                 var imagePipeline = ImagePipelineFactory.Instance.GetImagePipeline();
                 var image = default(BitmapSource);
                 var uri = new Uri(source);
+                var builder = ImageRequestBuilder.NewBuilderWithSource(uri);
+                var imageData = GetImageData(view);
 
-                // Remote images
-                if (source.StartsWith("http:") || source.StartsWith("https:"))
+                // Apply postprocessor if Composition isn't supported.
+                if (imageData.BackgroundColor != null || 
+                    (imageData.TintColor != null && !EffectManager.IsCompositionSupported))
```

Comment:
Nope, it's ignored by the postprocessor if it's null.

https://github.com/Microsoft/react-native-windows/pull/1319/files#diff-1411f606f43af4e7125cf4ab3d01d1fb